scripts-dev/complement.sh

```diff
@@ -214,7 +214,7 @@ fi
 
 extra_test_args=()
 
-test_tags="synapse_blacklist,msc3874,msc3890,msc3391,msc3930,faster_joins"
+test_packages="./tests/csapi ./tests ./tests/msc3874 ./tests/msc3890 ./tests/msc3391 ./tests/msc3930 ./tests/msc3902"
```

Comment:
slightly unfortunate that you have to say `./tests` here all the time... but I guess this lets you specify test packages in some other repo too?

Reply:
Yes